_pages/bea/current.md

```diff
@@ -125,13 +125,13 @@ We are extremely grateful to our sponsors for the past workshops: in the recent
 
 - [Ekaterina Kochmar](https://www.cl.cam.ac.uk/~ek358/), University of Bath
 - [Jill Burstein](https://sites.google.com/site/jbursteinets/), Duolingo
-- [Andrea Horbach](https://www.ltl.uni-due.de/team/andrea-horbach/), University Duisburg-Essen
+- [Andrea Horbach](https://www.fernuni-hagen.de/forschung/schwerpunkte/d2l2/team/andrea.horbach.shtml), FernUniversität in Hagen
 - [Ronja Laarmann-Quante](https://www.ltl.uni-due.de/team/ronja-laarmann-quante), University of Duisburg-Essen
 - [Nitin Madnani](https://desilinguist.org/), Educational Testing Service
 - [Anaïs Tack](https://anaistack.github.io/), Stanford University
 - [Victoria Yaneva](http://www.victoriayaneva.info/), National Board of Medical Examiners
 - [Zheng Yuan](https://www.cl.cam.ac.uk/~zy249/), University of Cambridge
-- [Torsten Zesch](https://www.ltl.uni-due.de/team/torsten-zesch), University of Duisburg-Essen
+- [Torsten Zesch](https://www.ltl.uni-due.de/team/torsten-zesch), FernUniversität in Hagen
```

Comment:
Do we not want to change the URL for Torsten even though his affiliation changed?

Reply:
The lab homepage has not moved yet, so for the moment no.